vagrant/k8s/Vagrantfile

```diff
@@ -98,6 +98,18 @@ SCRIPT
 cluster = read_cluster_config
 create_etc_hosts(cluster)
 
+def customize(v)
+  # make all nics 'virtio' to take benefit of builtin vlan tag
+  # support, which otherwise needs to be enabled in Intel drivers,
+  # which are used by default by virtualbox
+  # changes the settings for the first 5 NICs, regardless of presence
+  1.upto(5) do |n|
```

Comment:
There are only 3 NICs in the original Vagrantfile with nictype virtio. I don't fully understand how these translate to inside the VM - Does it create 5 NICs with these properties or does it mention that apply this property to the first 5 NICs in the VM?

Also, as I understand this, the order of the NICs matter and it's hardly coupled with the quagga VMs. Did you test it with L3? Lets not change the ordering of the NICs.